pkg/appconsts/global_consts.go

```diff
@@ -14,16 +14,16 @@ import (
 // They can not change throughout the lifetime of a network.
 const (
 	// NamespaceVersionSize is the size of a namespace version in bytes.
-	NamespaceVersionSize = ns.NamespaceVersionSize
+	NamespaceVersionSize = shares.NamespaceVersionSize
 	// NamespaceVersionMaxValue is the maximum value a namespace version can be.
 	// This const must be updated if NamespaceVersionSize is changed.
 	NamespaceVersionMaxValue = math.MaxUint8
 
 	// NamespaceIDSize is the size of a namespace ID in bytes.
-	NamespaceIDSize = ns.NamespaceIDSize
+	NamespaceIDSize = shares.NamespaceIDSize
```

Comment:
[suggestion] Now that the `share` package will have its own `go.mod`, it might be more logical to define these constants e.g., `NamespaceIDSize` at the app level (where the `share` package is used and where the core logic of app resides). We can then pass them as parameters to the relevant structs and methods within the `share` package.